2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +398,14 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+friends.forEach((friend)=>{
+  let isColleage = friend.colleagues
+    .map((colleagueName) => colleagueName.name)
+    .includes("Stacie Villarreal");
+    if(isColleage){
+      friendsWhoAreColleaguesOfStacie.push(`${friend.name.first} ${friend.name.last}`);
+    }
+})
 /*
```

Comment:
Nice way of doing it. but I think it would be better if you have used the const instead of let. 
++++ good way of using arrow function along  with the array methods. which is better than using for loop ++++